gef.py

```diff
@@ -2012,6 +2017,9 @@ def gdb_disassemble(start_pc: int, **kwargs: int) -> Generator[Instruction, None
     arch = frame.architecture()
 
     for insn in arch.disassemble(start_pc, **kwargs):
+        assert isinstance(insn["addr"], int)
```

Comment:
this is mostly for the pylance, as the gdb.pyi simply uses a `dict[str, object]` for instruction